### PR TITLE
Feature/san akel/port nccs discover

### PR DESCRIPTION
**Pull request by sanAkel**
## PR checklist
- [x] Short (1 sentence) summary of your PR: 
    Additions to be able to build (and run) on NASA's [NCCS discover cluster](https://www.nccs.nasa.gov/). Also see following `additional information`.
- [x] Developer(s): 
    [Santha Akella](https://github.com/sanAkel), [Matthew Thompson](https://github.com/mathomp4)
- [x] Suggest PR reviewers from list in the column to the right.
@eclare108213, @apcraig thanks for reviewing. I can't get the labels to work, I guess it should be `Porting`?
- [x] Please copy the PR test results link or provide a summary of testing completed below.
    See attached log file [icepack.runlog.220815-190420.txt](https://github.com/CICE-Consortium/Icepack/files/9340531/icepack.runlog.220815-190420.txt)
- How much do the PR code changes differ from the unmodified code? 
    - [ ] bit for bit
    - [ ] different at roundoff level
    - [ ] more substantial
    - [x] N/A; scope of this is completely within `configuration/scripts/`, no _science_ code has been touched.
- Does this PR create or have dependencies on CICE or any other models?
    - [ ] Yes
    - [x] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [x] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/.)
    - [ ] Yes
    - [x] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [x] No 
- [x] Please provide any additional information or relevant details below:
   This PR is purely _technical_: adds the capability to build and run standalone Icepack on another machine, namely [NCCS discover cluster](https://www.nccs.nasa.gov/) which is not currently supported in [present release](https://github.com/CICE-Consortium/Icepack/releases/tag/Icepack1.3.2). The additions followed Sec 3.2.2 `Porting` of [Icepack Documentation.](https://buildmedia.readthedocs.org/media/pdf/cice-consortium-icepack/main/cice-consortium-icepack.pdf)

### CODE REVIEW

**Review by apcraig — APPROVED** [review]

**Comment by phil-blain** [comment]
small nit, but the title of the PR becomes the title of the commit message once the PR is merged. Thus it would be nice for it to be more explicit, something like:

> machines: port to NASA NCCS Discover

or similar.

**Comment by sanAkel** [comment]
> small nit, but the title of the PR becomes the title of the commit message once the PR is merged. Thus it would be nice for it to be more explicit, something like:
> 
> > machines: port to NASA NCCS Discover
> 
> or similar.

Sorry @phil-blain somehow I over looked it. (I did it submit this PR while helping my kid with his homework! ☹️ )

**Comment by phil-blain** [comment]
No worries, Tony was too eager to click the Merge button :)

**Pull request merged**
